### PR TITLE
[TwigBundle] Add missing CHANGELOG entries and remove deprecated templates

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Follow up https://github.com/symfony/symfony/pull/32696, 
Require merge 4.4 into master after https://github.com/symfony/symfony/pull/32695 to make WebProfilerBundle work properly in master.

### CODE REVIEW

**Review by Tobion — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php [hunk 1]
```diff
@@ -190,9 +191,9 @@ public function testTwigLoaderPaths($format)
             ['namespaced_path1', 'namespace1'],
             ['namespaced_path2', 'namespace2'],
             ['namespaced_path3', 'namespace3'],
-            [__DIR__.'/Fixtures/templates/bundles/TwigBundle', 'Twig'],
-            [realpath(__DIR__.'/../..').'/Resources/views', 'Twig'],
-            [realpath(__DIR__.'/../..').'/Resources/views', '!Twig'],
+            [__DIR__.'/Fixtures/templates/bundles/AcmeBundle', 'Acme'],
+            [realpath(__DIR__.'/AcmeBundle').'/Resources/views', 'Acme'],
```
[the remark this answers is not in the document]
Comment (yceruto) [reply src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php]: Let's wait for https://github.com/symfony/symfony/pull/32740

[the remark this answers is not in the document]
Comment (Tobion) [reply src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php]: merged to master

[the remark this answers is not in the document]
Comment (yceruto) [reply src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php]: Rebased.

**Comment by Tobion** [comment]
Why did you drop the removal of the deprecated /views?

**Comment by yceruto** [comment]
Because I forgot to send the second commit :)

**Review by Tobion — APPROVED** [review]

**Comment by Tobion** [comment]
Thank you @yceruto.

**Pull request merged**